src/iocore/cache/CacheVC.h

```diff
@@ -281,8 +281,7 @@ struct CacheVC : public CacheVConnection {
   int                       fragment;
   int                       scan_msec_delay;
   CacheVC                  *write_vc;
-  char                     *hostname;
-  int                       host_len;
+  std::string_view          hostname;
```

Comment:
@cmcfarlen This is part of "Region C". Can we change this? IIRC, I broke cache compatibility by changing type in this region.

Reply:
@bneradt do you remember what you faced with `time_t pin_in_cache`?
https://github.com/apache/trafficserver/pull/11013

Reply:
If changing type of member of `CacheVC` breaks cache compatibility, I afraid this change. Because the size are different. The underlaying types are `char *` and `size_t`, I guess.
```
(lldb) p sizeof(char *)
(unsigned long) 8
(lldb) p sizeof(int)
(unsigned long) 4
(lldb) p sizeof(std::string_view)
(unsigned long) 16
```

Let me clarify the symptom, if cache compatibility is broken, we'll face warnings when new ATS read old cache from disk, right?

Reply:
I remember we did back and forth around `pin_in_cache`. A change of `Doc` obviously breaks compatibility, but change in the CacheVC was not actually?
[back and forth"](https://ell.stackexchange.com/questions/189720/back-and-forward-or-back-and-forth)